storage/innobase/xtrabackup/test/t/xbcloud.sh

```diff
@@ -15,6 +15,30 @@
 
 . inc/common.sh
 
+### kill all running queries
+function kill_sleep_queries()
+{
+  run_cmd $MYSQL $MYSQL_ARGS --force --batch  test <<EOF
+  select concat('KILL ',id,';') from information_schema.processlist
+  where user='root' and info like '%SLEEP%' and  time > 1 into outfile '$MYSQLD_TMPDIR/killall.sql';
+  source $MYSQLD_TMPDIR/killall.sql;
+EOF
+  rm -f $MYSQLD_TMPDIR/killall.sql
+}
+
+function wait_for_process_to_start() {
+  local pid=$1
+  vlog "wait for $pid to start "
+  while  ! kill -0 $pid >/dev/null 2>&1
+  do
+    vlog "waiting for pid $pid to start"
+    sleep 1
+  done
+}
```

Comment:
rename this to something more clear like, `wait_for_pid` and move it to common.sh